todo.py

```
@@ -0,0 +1,135 @@
+#!/usr/bin/env python3
+import sqlite3
+import argparse
+import sys
```

Ruff (0.8.2)

`sys` imported but unused

Remove unused import: `sys`

(F401)